pkg/drawio/layout.go

```diff
@@ -138,33 +138,36 @@ func sortGroupSquareBySize(groups []SquareTreeNodeInterface) []SquareTreeNodeInt
 //	         - else its visibility is connectedPoint
 func (ly *layoutS) calcGroupsVisibility(subnet SquareTreeNodeInterface) {
 	sortedBySizeGroups := sortGroupSquareBySize(subnet.(*SubnetTreeNode).groupSquares)
-	iconSquareGroups := map[IconTreeNodeInterface]map[SquareTreeNodeInterface]bool{}
+	iconShownSquareGroups := map[IconTreeNodeInterface]map[SquareTreeNodeInterface]bool{}
 	for _, groupS := range sortedBySizeGroups {
 		group := groupS.(*GroupSquareTreeNode)
-		if len(group.groupedIcons) == len(subnet.(*SubnetTreeNode).NIs()) {
-			group.setVisibility(theSubnet)
-			continue
-		}
 		groupedIconsFormerGroups := map[SquareTreeNodeInterface]bool{}
```

Comment:
`groupedIconsFormerGroups` doesn't really have to be a map. Consider converting it to a slice.